lib/kubernetes-deploy/logger.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+require 'logger'
+
+module KubernetesDeploy
+  module Logger
+    def self.included(base)
```

Comment:
i think it's a little overkill for now but if we start including other embeddable modules it would be worth it.